### DISCUSSION TITLE
PDF report

### DISCUSSION THREAD

**Discussion by haianhle** [vdemichev/DiaNN#540] [Q&A]
Hi Vadim, in the GUI version of DIANN there is a "Generate PDF report" check box. It appears that when I run DIANN from command line the PDF report is not generated. Is there a way to generate the PDF report from cmd line? I tried unchecking this "Generate PDF report" box to find out what the command might be but nothing changed. Thank you so much!

Best wishes,
Anh

**Comment by vdemichev** [comment]
Hi Anh,

When you use this feature in the GUI, it prints in the log, after the analysis is finished, the exact command used to invoke the plotting executable. Just need to reproduce that.

Best,
Vadim

**Comment by haianhle** [comment]
I swear I didn’t see a difference 😅 but I will check again thank you!

Anh
